modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc

```diff
@@ -32,27 +32,27 @@ Kata is installed on all worker nodes by default. If you want to install `kata`
 . Copy and paste the following manifest into the *YAML view*:
 
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: kataconfiguration.openshift.io/v1
 kind: KataConfig
 metadata:
   name: cluster-kataconfig
 spec:
-  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
+  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:{sandboxed-containers-version}.0
```

Comment:
Should the two references in this module be sandboxed-containers-version-z?

Reply:
Same comment as above.